qrimatic/api/cron.go

```diff
@@ -127,3 +130,16 @@ func (s *Server) RunHandler(w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte("not finished"))
 	// c.runWorkflow(r.Context(), nil)
 }
+
+// WorkflowListHandler returns a list of WorkflowInfos, which include workflows
+// that do and do not include runs
+func (s *Server) WorkflowListHandler(w http.ResponseWriter, r *http.Request) {
+	data, err := s.sched.ListWorkflowInfos(context.Background(), s.inst, 25, 0)
```

Comment:
use the request context instead of a background context, and flip the limit/offset to `offset`,`limit`:

```suggestion
	data, err := s.sched.ListWorkflowInfos(r.Context(), s.inst, 0, 25)
```